TLCGen.Dependencies/TLCGen.Dependencies/Plugins/ITLCGenHasSpecification.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using TLCGen.Models;
+
+namespace TLCGen.Plugins
+{
+    public enum SpecificationParagraphType
+    {
+        Body, Header1, Header2, TableHeader, Spacer
+    }
+
+    public enum SpecificationSubject
+    {
+        Intro,
+        Structure,
+        DynHiaat,
+        AFM,
+        GebruikersPlugin1,
+        GebruikersPlugin2
+    }
+
+    public abstract class SpecificationDataBase
+    {
+    }
+
+    public class SpecificationParagraph : SpecificationDataBase
+    {
+        // Body / Header1 / Header2 / etc <- dit is domeinspecifiek
+        public SpecificationParagraphType Type { get; set; }
+        // De tekst, al dan niet geformat
+        public string Text { get; set; }
+    }
+
+    public class SpecificationTable : SpecificationDataBase
+    {
+        public List<List<string>> TableData { get; set; }
+    }
+
+    public class SpecificationTable2 : SpecificationDataBase
```

Comment:
Wat is de reden van een tweede class? Deze is qua ontwerp identiek aan de eerste

Reply:
De reden weet ik niet (meer), hij lijkt als enige in de AFM plugin gebruikt te worden.
Aangepast en xxxxTable2 verwijderd.